pkg/base/helm_v3.go

```diff
@@ -0,0 +1,49 @@
+package base
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/releaseutil"
+)
+
+func renderHelmV3(chartName string, chartPath string, vals map[string]interface{}, renderOptions *RenderOptions) (map[string]string, error) {
+	cfg := &action.Configuration{
+		Log: renderOptions.Log.Debug,
+	}
+	client := action.NewInstall(cfg)
+	client.DryRun = true
+	client.ReleaseName = chartName
+	client.Replace = true
+	client.ClientOnly = true
+	// client.IncludeCRDs = includeCrds
```

Comment:
not sure what to do here

Reply:
I'd think we'd generally have to include CRDs, right?